apps/pie-docs/src/designers/how-we-support-you.md

```diff
@@ -79,10 +79,10 @@ As a result of that, we’re often asked how PIE designers are usually involved
     iconName: "check-circle-filled",
     iconFill: "support-positive",
     items: [
-        "Provide guidance and support with foundations, components and patterns.",
-        "Create, update and maintain our foundations, components and patterns.",
-        "Write documentation about the system’s elements behaviour and interactions.",
-        "Assist business area designers when they create their own components."
+        "Provide guidance and support with foundations, components and patterns",
+        "Create, update and maintain our foundations, components and patterns",
+        "Write documentation about the system’s elements behaviour and interactions",
```

Comment:
```suggestion
        "Write documentation about the system’s elements’ behaviours and interactions",
```

Reply:
@LTurns I know you've deleted your comment now, but it was a little bit of a strange sentence. I think I'd have reworded it to avoid the need for apostrophes altogether, something like this:
```
"Write documentation about the behaviours and interactions of the design system elements",
```
(Optionally, "design system's elements")

But keeping it as close to the original copy as possible, you do need the `'s` because the system has multiple elements.

In fact I think the original sentence missing the word "design" makes it harder to understand what it's talking about.

Reply:
yep makes sense, I deleted it because I'd misread the sentence and realised you do actually need the `'` there 😄 

I prefer how you've written it, I think it reads better that way. Appreciate it's quite different to the copy though